octobot_trading/modes/channel/abstract_mode_producer.py

```diff
@@ -407,15 +407,19 @@ async def cancel_symbol_open_orders(self, symbol, side=None) -> bool:
         """
         cancel_loaded_orders = self.get_should_cancel_loaded_orders()
         cancelled = False
+        failed_to_cancel = False
         if self.exchange_manager.trader.is_enabled:
             for order in self.exchange_manager.exchange_personal_data.orders_manager.get_open_orders(symbol=symbol):
                 if (
                     not (order.is_cancelled() or order.is_closed())
                     and (cancel_loaded_orders or order.is_from_this_octobot)
                     and (side is None or (side is order.side))
                 ):
-                    cancelled = await self.trading_mode.cancel_order(order) and cancelled
-        return cancelled
+                    if await self.trading_mode.cancel_order(order):
+                        cancelled = True
+                    else:
+                        failed_to_cancel = True
+        return cancelled and not failed_to_cancel
```

Comment:
Why not just return `not failed_to_cancel` ?

Reply:
we want to be sure we cancelled at least one order